cmd/mdatagen/internal/templates/logs_test.go.tmpl

```diff
@@ -80,3 +80,50 @@ func TestLogsBuilderAppendLogRecord(t *testing.T) {
 	assert.Equal(t, pcommon.ValueTypeStr, sl.LogRecords().At(1).Body().Type())
 	assert.Equal(t, "the second log record", sl.LogRecords().At(1).Body().Str())
 }
+
+{{- if .Events }}
+func TestLogsBuilder(t *testing.T) {
```

Comment:
This test doesn't do anything yet right? Let's not generate it for now then

Reply:
removed.